src/monitor-view-events.js

```diff
@@ -0,0 +1,53 @@
+// DOM Refresh
```

Comment:
So I'm not sure this is a util either..  Is bind-events and triggerMethod a util?